### ISSUE TITLE
WIndows vista  -POL hang and crash

### ISSUE THREAD

**Issue by Cloudstrafie** [Windower/Issues#906]
https://www.dropbox.com/s/a5exklbglrvpb07/pol.exe.7032.dmp?dl=0

https://www.dropbox.com/s/n2jmi8g6nsafl1f/pol.exe.7132.dmp?dl=0

https://www.dropbox.com/s/a118vf9a08r63mq/pol.exe.8336.dmp?dl=0

OS is windows vista, Windower starts up then after  selecting profile POL crashes on start up.

This is for a friend who is on vista and needed help with crash dumps uploaded

**Comment by cairface** [comment]
Hi!

Is this still happening? I unfortunately no longer have the symbols for this hook (not sure what happened, sorry!), so I can't debug with those dumps. If it's still happening on the dev branch, if you provide a dump for me, I will look into it. Sorry for the inconvenience.

**Comment by Cloudstrafie** [comment]
I'm not entirely sure I didn't have Internet this morning when I woke up(thank you comcast) I'll have to check once I get home this evening. But I was having the same issue on 4.3 as well as the dev build but I'll try both once I get home.

**Comment by Cloudstrafie** [comment]
Yes currently this is still an issue,Ill try and have my friend upload a dmp file as soon as i can

**Comment by Rockfist** [comment]
I have the same problem with my windower on Vista 32 bits... always POL Crash.. no idea what to do.. 
tried everything that possibly could be done. Would really like to see a solution posted !

**Comment by garnerv** [comment]
I'm having the same issue here. Crash immediately on launch of profile on windower.

**Comment by Nifim** [comment]
Vista problem, if it is still around not sure we are going to do much about it.
